torch/csrc/generic/utils.h

```diff
@@ -1,33 +0,0 @@
-#ifndef TH_GENERIC_FILE
-#define TH_GENERIC_FILE "torch/csrc/generic/utils.h"
-#else
-
-#if defined(TH_REAL_IS_HALF)
-#define GENERATE_SPARSE 0
-#else
-#define GENERATE_SPARSE 1
-#endif
-
-struct THPStorage;
-struct THSPTensor;
```

Comment:
I think this `THSPTensor` thing is no longer used at all, and there are other files related to it that could be cleaned up. I'll find out for sure and remove that stuff in another PR if so.